CaloConditions/src/CalCalib.cc

```diff
@@ -0,0 +1,17 @@
+#include "Offline/CaloConditions/inc/CalCalib.hh"
+
+#include <sstream>
+
+using namespace std;
+
+namespace mu2e {
+
+  void CalCalib::print( ostream& out) const{
+    out << "ADCMeV "
+      <<  ( _calpar->ADC2MeV() ) << endl;
```

Comment:
This reference also need to be fixed, thanks

Reply:
OK, I've removed this file, I dont think its needed yet, if at all